### DISCUSSION TITLE
Where is the metadata for the incompatible (with the diffusers backend) extensions?

### DISCUSSION THREAD

**Discussion by lbeltrame** [vladmandic/sdnext#2677] [Q&A] [answered]
As the subject says. Is it kept alongside the extension list metadata or elsewhere? I ask because I want to submit a PR to add the "cutoff" extension (the one that works with SD.Next, anyway), because it spams the logs with (harmless) errors when using the diffusers backend (at least with SDXL, I haven't seen with SD, but I expect so, as the backend is quite different). It's mostly pointless with SDXL at least anyway (at least the models I've tested have basically zero color bleeding).

**Comment by vladmandic** [comment, marked as answer]
its in <https://github.com/vladmandic/sd-data/tree/main/lists>
and format is documented in <https://github.com/vladmandic/sd-data/blob/main/README.md>

if by any chance you plan to submit more than one entry, you can create a PR to create a new (your) file in that folder.
otherwise, do a PR to modify an existing one.

**Comment by lbeltrame** [comment]
An additional question: is that metadata then used to disable installed extensions if incompatible (e.g. ControlNet with diffusers)?

**Reply by vladmandic** [reply]
No, that list is a hardcoded one.
Metadata is currently only used as indicator in extensions tab.
